swagger_parser/lib/src/parser/parser.dart

```diff
@@ -29,8 +29,7 @@ class OpenApiJsonParser {
         return;
       }
     }
-    if (_jsonContent.containsKey('swagger') &&
```

Comment:
It is necessary to roll back the changes to this file (parser.dart). Because in the project the length of the lines is 80 characters.